docs/en/sql-reference/functions/ext-dict-functions.md

```diff
@@ -7,226 +7,188 @@ sidebar_label: Dictionaries
 
-``` text
-1,1,'1'
-2,2,'2'
-3,3,'3'
-```
-
-The first column is `id`, the second is `c1`, the third is `c2`.
-
-Configure the dictionary:
-
-``` xml
-<clickhouse>
-    <dictionary>
-        <name>ext-dict-mult</name>
-        <source>
-            <file>
-                <path>/path-to/ext-dict-mult.csv</path>
-                <format>CSV</format>
-            </file>
-        </source>
-        <layout>
-            <flat />
-        </layout>
-        <structure>
-            <id>
-                <name>id</name>
-            </id>
-            <attribute>
-                <name>c1</name>
-                <type>UInt32</type>
-                <null_value></null_value>
-            </attribute>
-            <attribute>
-                <name>c2</name>
-                <type>String</type>
-                <null_value></null_value>
-            </attribute>
-        </structure>
-        <lifetime>0</lifetime>
-    </dictionary>
-</clickhouse>
-```
-
-Perform the query:
+**Syntax**
 
-``` sql
-SELECT
-    dictGet('ext-dict-mult', ('c1','c2'), number + 1) AS val,
-    toTypeName(val) AS type
-FROM system.numbers
-LIMIT 3;
+```sql
+dictGetOrDefault('dict_name', attr_names, id_expr, default_value_expr)
 ```
 
-``` text
-┌─val─────┬─type──────────────────┐
-│ (1,'1') │ Tuple(UInt8, String)  │
-│ (2,'2') │ Tuple(UInt8, String)  │
-│ (3,'3') │ Tuple(UInt8, String)  │
-└─────────┴───────────────────────┘
-```
+**Arguments**
+
+- `dict_name` — Name of the dictionary. [String literal](../../sql-reference/syntax.md#syntax-string-literal).
+- `attr_names` — Name of the column of the dictionary, [String literal](../../sql-reference/syntax.md#syntax-string-literal), or tuple of column names, [Tuple](../../sql-reference/data-types/tuple.md)([String literal](../../sql-reference/syntax.md#syntax-string-literal)).
+- `id_expr` — Key value. [Expression](../../sql-reference/syntax.md#syntax-expressions) returning dictionary key-type value or [Tuple](../../sql-reference/data-types/tuple.md)-type value depending on the dictionary configuration.
+- `default_value_expr` — Values returned if the dictionary does not contain a row with the `id_expr` key. [Expression](../../sql-reference/syntax.md#syntax-expressions) or [Tuple](../../sql-reference/data-types/tuple.md)([Expression](../../sql-reference/syntax.md#syntax-expressions)), returning the value (or values) in the data types configured for the `attr_names` attribute.
+
+**Returned value**
 
-**Example for range key dictionary**
+If ClickHouse parses the attribute successfully in the [attribute’s data type](../../sql-reference/dictionaries/index.md#dictionary-key-and-fields#ext_dict_structure-attributes), this function returns the value of the dictionary attribute that corresponds to `id_expr`.
 
-Input table:
+If there is no the key, corresponding to `id_expr`, in the dictionary, then returns the value passed as the `default_value_expr` parameter.
+
+**Implementation details**
+
+None.
+
```

Comment:
If none, please, remove this section

Reply:
I disagree. Pointing out that there aren't any specific implementation details is useful. Otherwise it could look like we just forgot to put the details in (which has happened all throughout our docs).